src/Message/CancelSubscriptionRequest.php

```diff
@@ -160,14 +181,22 @@ public function getData()
             throw new InvalidRequestException('Either the subscription id or reference is required.');
         }
 
-        return array(
+        $data = array(
             'autobill' => $subscription,
             'action' => $this->getFunction(),
-            'disentitle' => false,
-            'force' => true,
             'settle' => false,
             'sendCancellationNotice' => false,
-            'cancelReasonCode' => $this->getCancelReason()
+            'cancelReasonCode' => $this->getCancelReason(),
+            'force' => true,
+            'disentitle' => false
         );
+
+        $coerce = $this->getCoercion();
+        if ($coerce) {
```

Comment:
So this is really a shortcut so that we don't need to do the following right?

```PHP
        $response = $gateway->cancelSubscription([
            'subscriptionReference' => $this->vid,
            'force'  => true,
             'disentitle' => true
        ])->send();
```

Reply:
Correct!